src/main/java/edu/wpi/grip/ui/controllers/MainWindowController.java

```diff
@@ -38,6 +42,14 @@
                 + context.getSubscriber() + " to " + context.getSubscriberMethod(), exception).printStackTrace(System.err);
     });
 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Perhaps 'eventHistory' could be replaced by a local variable.](https://www.codacy.com/app/WPIRoboticsProjects/GRIP/file/1275438274/issues/source?bid=2943107&fileBranchId=2943115#l45)